README.md

```diff
@@ -1,3 +1,5 @@
+[![NuGet Version](https://img.shields.io/nuget/vpre/OpenTelemetry.Extensions.AWS)](https://www.nuget.org/packages/OpenTelemetry.Extensions.AWS)[![NuGet Downloads](https://img.shields.io/nuget/dt/OpenTelemetry.Extensions.AWS)](https://www.nuget.org/packages/OpenTelemetry.Extensions.AWS)
```

Comment:
I don't think we should have just the OpenTelemetry.Extensions.AWS badge. We could have all the badges, but that would probably be cluttered. Perhaps a table of components with badges after the initial section?

Reply:
And we have badges on dedicated README files https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/1ecd3fdc3a0c9855670165d160661dddf0508ff9/src/OpenTelemetry.Extensions.AWS

I doubt that we need to duplicate.